ansible/base.yaml

```diff
@@ -21,10 +21,9 @@
         name:
           - git
           - gcc
-          - golang-1.19-go
+          - golang-1.21-go
           - prometheus
           - prometheus-node-exporter
-          - ntp
```

Comment:
Why was ntp removed?
We added it to make sure the latencies we are calculating are accurate